api/src/main/java/org/openmrs/Provider.java

```diff
@@ -91,6 +95,35 @@ public void setIdentifier(String identifier) {
 	public String getIdentifier() {
 		return identifier;
 	}
+
+
+	/**
```

Comment:
Can you do proper javadoc for these methods? Here is an example: https://github.com/openmrs/openmrs-core/commit/f5132a1dda4a6a8fcbfe06a32dcbbe4b7a0a270d

Reply:
Done